HorseFacts.CatFactsApiPlugin.Tests/CatFactsApiGateway.Tests.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Linq;
+using FluentAssertions;
+using HorseFacts.CatFactsApiPlugin.Gateways;
+using HorseFacts.Core.Domain;
+using NUnit.Framework;
+using WireMock.Logging;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+using WireMock.Server;
+using WireMock.Settings;
+
+namespace Tests
+{
+    public class CatFactsApiGatewayTests
+    {
+        private FluentMockServer _server;
+
+        [SetUp]
+        public void Setup()
+        {
+            _server = FluentMockServer.Start(new FluentMockServerSettings());
+        }
+
+        [TearDown]
+        public void Teardown()
+        {
+            _server.Stop();
+        }
+
+        [Test]
+        [TestCase("Cats eat hay")]
```

Comment:
I'm not too sure about this one 🤔

Reply:
Sorry, I don't know much about cats, I only know horses.

Reply:
This is acceptable